src/loc/secrets/RemoveSecretDialog.tsx

```diff
@@ -0,0 +1,55 @@
+import { Secret } from "@logion/client";
+import Dialog from "../../common/Dialog";
+import { useForm } from "react-hook-form";
+import { useCallback } from "react";
+import Icon from "../../common/Icon";
+import ViewableSecret from "./ViewableSecret";
+import "./RemoveSecretDialog.css"
+
+export interface Props {
+    secret: Secret | undefined;
+    onRemoveSecret: (secret: Secret) => void;
+    onCancel: () => void;
+}
+
+export default function RemoveSecretDialog(props: Props) {
+    const { handleSubmit, reset } = useForm<{}>();
```

Comment:
Not sure it makes sense to use `react-hook-form` in this case.

Reply:
Indeed. It's removed, thanks for the suggestion and the support.